docs/content/en/docs/references/api/grpc.md

```diff
@@ -828,6 +855,9 @@ Enum for Suggestion codes
 | ADDRESS_NODE_NOT_READY | 406 | Node not ready error |
 | ADDRESS_FAILED_SCHEDULING | 407 | Scheduler failure error |
 | CHECK_HOST_CONNECTION | 408 | Cluster Connectivity error |
+| START_MINIKUBE | 501 | Minikube Suggestions |
+| UNPAUSE_MINIKUBE | 502 |  |
+| OPEN_ISSUE | 900 | Open Issue Suggestion |
```

Comment:
The doc comments for these items need to be fixed up

Reply:
yes. good point.